R/helpers.R

```diff
@@ -237,3 +237,28 @@ view_url <- function(..., open = interactive()) {
   }
   invisible(url)
 }
+
+bump_version <- function(ver, choice) {
```

Comment:
I'd stick with same vocabulary and input values. So "which" instead of "choice" and `c("major", "minor", "patch", "dev")` as possible values.